components/avatar.css

```diff
@@ -0,0 +1,28 @@
+.avatar-xsm {
+	width: 3rem;

```

Comment:
Could have used a standard name like bootstrap and other libraries add for extra small screens.